Server/ChatService/routes/chat.py

```
@@ -62,6 +62,54 @@ async def mark_user_online_instantly(user_id: str):
+        while True:
+            await asyncio.sleep(30)  # Ping every 30 seconds
+            
+            # Check if connection is still active (has received data in last 60 seconds)
+            last_activity = get_last_activity()
+            current_time = asyncio.get_event_loop().time()
+            if current_time - last_activity > 60:
+                print(f"[chat.ws] Connection timeout for user {user_id} (no activity for {current_time - last_activity:.1f}s)")
+                try:
+                    await websocket.close(code=1008)  # Policy violation (timeout)
+                except Exception:
+                    pass
```

Ruff (0.14.5)

`try`-`except`-`pass` detected, consider logging the exception

(S110)